### PR TITLE
Extend support to existing Web5 tutorial examples.

### PR DESCRIPTION
**Pull request by acekyd**
In this update:

- Added support to download example subdirectories in the docs.
- Extended the terminal options to support the extra parameters.

### CODE REVIEW

File: actions/example.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { downloadTemplate } from 'giget';
+import fs from 'fs-extra';
+
+export async function example(project, context) {
```
Comment (kirahsapong) [inline actions/example.js]: should we name this `cloneExample` ?

File: actions/example.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+export function getExampleTarget(project) {
+    const pathToExamples = 'TBD54566975/developer.tbd.website/examples/tutorials/';
+    return `github:${pathToExamples}${project}`;
+}
+
+export default async function copyExample(project, context) {
+    const exampleTarget = getExampleTarget(project);
+    //Copy
+    try {
+        const response = await downloadTemplate(exampleTarget, {
```
Comment (kirahsapong) [inline actions/example.js]: [nit] should we remove the assignment to `response`? I think we dont use it anywhere

```suggestion
        await downloadTemplate(exampleTarget, {
```

File: index.js [hunk 3]
```diff
@@ -25,103 +26,118 @@ if (argv.e && argv.endpoints) {
...
 
+if (argv.ex && argv.example) {
+  console.error(`Error: illegal options: specify either -ex or --example, not both.`)
+  process.exit(1)
+}
+
 let template = argv.t || argv.template;
 let sync = argv.s || argv.sync;
 let endpoints = argv.e?.join(', ') || argv.endpoints;
+let example = argv.ex || argv.example;
```
Comment (kirahsapong) [inline index.js:37]: should we add a console message here in the case the user specifies example, template, sync, and endpoints? 
```suggestion
if (example && (template || sync || endpoints)) { 
  console.error(`Example app ${example} was selected, but isn't configurable. Other flags will be ignored.`) 
} 
```

**Review by kirahsapong — APPROVED** [review]
This is awesome 🔥

File: actions/example.js (hunk 1 above)
Comment (acekyd) [reply actions/example.js]: That would probably be easier. Left it to be easy to correlate with future actions but we might as well combine action with target e.g `cloneExample` or `setupTbdex` etc in future. Will make a change

File: actions/example.js (hunk 2 above)
Comment (acekyd) [reply actions/example.js]: Yup! Had that for logging. Removing.

File: index.js (hunk 3 above)
Comment (acekyd) [reply index.js:37]: Ah yes that's genius! Thanks

Comment (kirahsapong) [reply index.js:37]: uh oh im sorry! i accidentally made that a suggested change 🤦‍♀️  we should add `let example = argv.ex || argv.example;` back in there, hahaha

Comment (acekyd) [reply index.js:37]: Haha no worries. caught that already. Should be ready to go now. ✨

**Pull request merged**